### ISSUE TITLE
[BUG] Twitter timeout error on firefox

### ISSUE THREAD

**Issue by Sue9445** [microsoft/playwright#4297]
Hi,

When i trying to navigate to Twitter in Firefox, I'm getting timeout exceeded errors.Its working the other browsers.

https://try.playwright.tech/?s=q5t4t4p

**Comment by pavelfeldman** [comment]
I can repro

**Comment by MohsinA1i** [comment]
> When I trying to navigate to Twitter in Firefox, I'm getting timeout exceeded errors. Its working the other browsers.

I've experienced similar problems. As a temporary fix instead of using
```
await page.goto('https://www.twitter.com', {waitUntil: "load"}); 
```
I instead use
```
await this.page.evaluate(() => window.location.href = 'https://www.twitter.com');
await this.page.waitForFunction(() => document.readyState == 'complete');
```

**Comment by aslushnikov** [comment]
For the record: This happens due to cross-process navigation. I'm working on a fix!

**Comment by Sue9445** [comment]
Is this bug will be fixed in v1.7? @pavelfeldman
